internal/service/device.go

```diff
@@ -5,14 +5,15 @@ import (
 
 	api "github.com/flightctl/flightctl/api/v1alpha1"
 	"github.com/flightctl/flightctl/internal/server"
+	"github.com/go-openapi/swag"
 	"github.com/google/uuid"
 	"gorm.io/gorm"
 	"k8s.io/apimachinery/pkg/labels"
 )
 
 type DeviceStoreInterface interface {
 	CreateDevice(orgId uuid.UUID, device *api.Device) (*api.Device, error)
-	ListDevices(orgId uuid.UUID, labels map[string]string) (*api.DeviceList, error)
+	ListDevices(orgId uuid.UUID, labels map[string]string, limit int, cont string) (*api.DeviceList, error)
```

Comment:
there will probably be additional parameters in the future, consider using struct parameters
```
type ListDeviceParams struct {
     Limit int
     Cont string
     ...
}
```

Reply:
Done